### ISSUE TITLE
copied directories are still symbolic links

### ISSUE THREAD

**Issue by mpoelchau** [NAL-i5K/Organism_Onboarding#83]
It looks like the dispatch workflow is still having trouble with copying directories.  I don't understand why - it looks like cwl is running the correct commands, and copying single files is not a problem.

Here's an example stderror from running the workflow:

```
[workflow 2other_species] starting step cp_seq
[step cp_seq] start
[job cp_seq] /tmp/X8sYGj$ cp \
    -r \
    seq \
    /app/data/BCM_i5k_pilot/fraocc/Focc_3.0/jbrowse/data/
Could not collect memory usage, job ended before monitoring began.
[job cp_seq] completed success
[step cp_seq] completed success
```
The command looks fine to me, and cwl says everything worked fine. But when I go to  /app/data/BCM_i5k_pilot/fraocc/Focc_3.0/jbrowse/, I see that the seq, tracks and names directories are symbolic links, and weren't copied:

```
$ ls -l
total 8
lrwxrwxrwx 1 mpoelchau i5k   21 Dec  6 09:07 bigwig -> ../../scaffold/bigwig
lrwxrwxrwx 1 mpoelchau i5k   22 Dec  6 09:10 names -> /tmp/Q9ggb_/data/names
lrwxrwxrwx 1 mpoelchau i5k   20 Dec  6 09:09 seq -> /tmp/b2CuuM/data/seq
-rw-rw-r-- 1 mpoelchau i5k 2061 Dec  6 09:09 trackList.json
-rw-rw-r-- 1 mpoelchau i5k 2061 Dec  6 09:08 trackList.json.bak-metadata
lrwxrwxrwx 1 mpoelchau i5k   23 Dec  6 09:09 tracks -> /tmp/47laFT/data/tracks
-rw-rw-r-- 1 mpoelchau i5k    0 Dec  6 09:08 tracks.conf
```

@HsiuKangHuang can you try to figure out what's going on here? I tried to search for similar issues, but don't have the time to explore them fully.

Possibly related:
- https://www.biostars.org/p/397240/#397685
- https://cwl.discourse.group/t/passing-reference-or-whole-directory-as-input/64/2
- https://www.commonwl.org/v1.0/CommandLineTool.html#Dirent
- https://www.biostars.org/p/225885/

**Comment by HsiuKangHuang** [comment]
Hi, Monica
I will check this on my computer again and see what I can do for this
problem.
For the goanna, I replaced "-num_threads" to "-t" and also removed "-b"
before "yes", but it still can't work.
The error message I got is :

Error: Too many positional arguments (1), the offending value: _exponly
Error:  (CArgException::eSynopsis) Too many positional arguments (1), the
offending value: _exponly
Command line argument error: Argument "archive". File is not accessible:
 `.asn'

Kelly

mpoelchau <notifications@github.com> 於 2019年12月6日 週五 下午12:27寫道：
